core/variant_parser.cpp

```diff
@@ -409,7 +409,7 @@ Error VariantParser::_parse_enginecfg(Stream *p_stream, Vector<String> &strings,
 	Token token;
 	get_token(p_stream, token, line, r_err_str);
 	if (token.type != TK_PARENTHESIS_OPEN) {
-		r_err_str = "Expected '(' in old-style godot.cfg construct";
+		r_err_str = "Expected '(' in old-style project.godot construct";
```

Comment:
@reduz Is all the engine.cfg compatibility stuff in VariantParser still relevant? IINM godot.cfg/project.godot now are using a valid VariantParser format?